test/e2e/tests/collector_pipeline/main_test.go

```diff
@@ -29,7 +29,12 @@ func TestMain(m *testing.M) {
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.ScannerImage),
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.CollectorImage),
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.VulnerableImage),
-		util.DeployEraserManifest(util.EraserNamespace, util.ProviderResource),
+		util.MakeDeploy(map[string]string{
```

Comment:
can we remove `DeployEraserManifest` if it's unused?

Reply:
It's already removed